### PR TITLE
Serge as executable

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
https://trello.com/c/VVNVWHAv

### CODE REVIEW

**Comment by IanMayo** [comment]
@lilitkarapetyan - are some of the new mode modules actually `dev-dependencies`?  Marking them as such _may_ give us the choice to minimise the install size (important when creating offline install CD).

File: pkgResolveErrors.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const fs = require("fs");
+// fix pouchdb-fauxton errr on pkg build
+/*
+> Error! Unexpected token (14:0)
+  C:\Users\User\Documents\serge\node_modules\pouchdb-fauxton\www\index.html
+*/
```
Comment (IanMayo) [inline pkgResolveErrors.js:6]: @lilitkarapetyan - just to help me learn, what is the bad token that `pkg` complains about?

Comment (lilitkarapetyan) [reply pkgResolveErrors.js:6]: @IanMayo pkg gives error on this html file, don't sure why (maybe trying to read it as js file), I can't figure out why it happens usualy html files uses for demo and do not included in main package. Only way that i found to fix it is that with leaveing it empty before pkg build

File: client/public/gconfig.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+window.G_CONFIG = {
+  REACT_APP_SERVER_PATH: null 
+}
```
Comment (IanMayo) [inline client/public/gconfig.js:3]: @lilitkarapetyan - what's the role of this variable?

File: bin.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const server = require("./server");
+const opn = require('opn');
+
+if(process.argv[2]) {
+  console.log(`running client with remote server "${process.argv[2]}"`);
+}
+else {
+  console.log("running client with local server");
+  console.log("(You can give the second parameter as the remote server URL)");
+}
```
Comment (IanMayo) [inline bin.js:10]: Could you extend the `readme`, to explain the second parameter is available, and to give an example of using it please?

**Comment by IanMayo** [comment]
@lilitkarapetyan - saw the "core configs" change.

But, isn't it then just hard-coded to those URLs?

What if it's on a different server, like in a VM?

**Pull request merged**